firefox-ios/Client/Frontend/Settings/AppearanceSettings/Generic views/GenericButtonCellView.swift

```diff
@@ -0,0 +1,44 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import SwiftUI
+import Common
+
+struct GenericButtonCellView: View {
```

Comment:
Looking at the video, it seems the button is either a mismatching color, or it should benefit from rounded corners? There's like a slight border on top and bottom as well and it confuses me 🤔 Also, should this be a component in the Component Library?

![Screenshot 2025-05-22 at 10 38 09 AM](https://github.com/user-attachments/assets/f50ae32c-8f50-4300-9565-ac252b4a72f3)